opensearch/config.php

```diff
@@ -201,11 +261,22 @@ public function getFacetOutput(array $results) {
 	* @param string $sort query result sort parameter
 	* @return string representation of response to client
 	*/
-	public function getResponse($type, array $constraints, $limit=null, $offset=0, $sort=null) {
-	
-		$query = $this->getSelectQuery($type, $constraints, $limit, $offset, $sort);		
-		$results = $this->sparqlSelect($query);				
-		return $this->getOutput($results, $type, $constraints, $limit, $offset);
+	public function getResponse($type, array $constraints, $limit=null, $offset=0, $sort=null)
+	{
+		$key = implode(func_get_args());
```

Comment:
add code documentation about what's going on here. Taking the list of arguments as an array. Imploding it into a string for the key. The value associated with the key is the html generated by getOutput, which takes the result of running the query and generating the html. Add comments above most lines of code